src/HOI4World/HoI4FocusTree.cpp

```diff
@@ -1137,33 +1137,39 @@ void HoI4FocusTree::addGPWarBranch(const HoI4::Country& home,
 		summitFocus->updateFocusElement(summitFocus->completionReward, "$IDEOLOGY", ideology);
 
 		events.createSummitNewsEvents(majorIdeologies);
-		auto eventIds = events.getSummitNewsEventsIds();
-		std::string hiddenEffect = "hidden_effect = {\n";
-		hiddenEffect += "\t\t\tif = {\n";
-		hiddenEffect += "\t\t\t\tlimit = { has_government = fascism }\n";
-		hiddenEffect += "\t\t\t\tevery_other_country = {\n";
-		hiddenEffect += "\t\t\t\t\tlimit = { is_major = yes }\n";
-		hiddenEffect += "\t\t\t\t\tnews_event = { id = " + eventIds["fascism"] + " }\n";
-		hiddenEffect += "\t\t\t\t}\n";
-		hiddenEffect += "\t\t\t}\n";
-		hiddenEffect += "\t\t}\n";
-		summitFocus->updateFocusElement(summitFocus->completionReward,
-			 "#FASC_NEWS",
-			 hiddenEffect,
-			 majorIdeologies.contains("fascism"));
-		hiddenEffect = "hidden_effect = {\n";
-		hiddenEffect += "\t\t\tif = {\n";
-		hiddenEffect += "\t\t\t\tlimit = { has_government = communism }\n";
-		hiddenEffect += "\t\t\t\tevery_other_country = {\n";
-		hiddenEffect += "\t\t\t\t\tlimit = { is_major = yes }\n";
-		hiddenEffect += "\t\t\t\t\tnews_event = { id = " + eventIds["communism"] + " }\n";
-		hiddenEffect += "\t\t\t\t}\n";
-		hiddenEffect += "\t\t\t}\n";
-		hiddenEffect += "\t\t}\n";
-		summitFocus->updateFocusElement(summitFocus->completionReward,
-			 "#COMM_NEWS",
-			 hiddenEffect,
-			 majorIdeologies.contains("communism"));
+		auto& eventIds = events.getSummitNewsEventsIds();
+		if (majorIdeologies.contains("fascism"))
+		{
+			std::string hiddenEffect = "hidden_effect = {\n";
+			hiddenEffect += "\t\t\tif = {\n";
+			hiddenEffect += "\t\t\t\tlimit = { has_government = fascism }\n";
+			hiddenEffect += "\t\t\t\tevery_other_country = {\n";
+			hiddenEffect += "\t\t\t\t\tlimit = { is_major = yes }\n";
+			hiddenEffect += "\t\t\t\t\tnews_event = { id = " + eventIds.at("fascism") + " }\n";
+			hiddenEffect += "\t\t\t\t}\n";
+			hiddenEffect += "\t\t\t}\n";
+			hiddenEffect += "\t\t}\n";
+			summitFocus->updateFocusElement(summitFocus->completionReward,
+				 "#FASC_NEWS",
+				 hiddenEffect,
+				 majorIdeologies.contains("fascism"));
```

Comment:
This can be removed as it's now checked in the 'if' condition and by default is false.

Reply:
Done!